Description: fixed import
 cherry picked fix for an import error during test
 .
 pipenv (11.9.0-1) unstable; urgency=medium
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Bastian Venthur <venthur@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2018-03-24

Index: pipenv/tests/test_vendor.py
===================================================================
--- pipenv.orig/tests/test_vendor.py
+++ pipenv/tests/test_vendor.py
@@ -8,7 +8,7 @@ from prettytoml.elements.metadata import
     WhitespaceElement, PunctuationElement, CommentElement
 )
 from prettytoml.elements.table import TableElement
-from pipenv.vendor.pipfile.api import PipfileParser
+from pipenv.patched.pipfile.api import PipfileParser
 
 
 def test_table():
